packages/oui-navbar/_mixins.less

```diff
@@ -94,9 +94,13 @@
   }
 
   .navbar-link_icon(
-    @icon-line-height: @oui-navbar-icon-line-height,
     @icon-size: @oui-navbar-icon-size,
-    @icon-spacing: @oui-navbar-icon-spacing,
+    @ending-icon-size: @oui-navbar-ending-icon-size,
```

Comment:
Careful by modifying an existing mixin. It could lead to a breaking change ;)

Reply:
You're right, I considered that `ui-kit` was only used by `ui-angular` but of course it could be wrong !